autonomy/cli/analyse.py

```diff
@@ -165,37 +168,47 @@ def parse_logs(file: Path) -> None:
         raise click.ClickException(str(e)) from e
 
 
-@abci_group.command(name="check-handlers")
-@click.argument(
-    "packages_dir",
-    type=click.Path(dir_okay=True, exists=True),
-    default=Path.cwd() / "packages",
-)
+@analyse_group.command(name="handlers")
+@pass_ctx
 @click.option(
-    "--skip",
+    "--common-handlers",
+    "-h",
     type=str,
-    default="abstract_abci",
-    help="Specify which skills to skip. Eg. skill_0,skill_1,skill_2",
+    default=[
+        "abci",
+    ],
+    help="Specify which handlers to check. Eg. -h handler_a -h handler_b -h handler_c",
+    multiple=True,
 )
 @click.option(
-    "--common",
+    "--ignore",
+    "-i",
     type=str,
-    default="abci",
-    help="Specify which handlers to check. Eg. handler_a,handler_b,handler_c",
+    default=[
+        "abstract_abci",
+    ],
```

Comment:
like I'm not well acquainted with click. But mutable default arguments give me the creeps